content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+> TEXT TO WRITE:
+> 
+> * talk about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app")
+>    * link to kubeflow-overview for more details
+>    * (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+> * short discussion about how Kubeflow is versioned, 
+>    * components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+> 
+> * short discussion about the two "paths" for installing kubeflow "XXXXX" and "YYYYY"
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+> TEXT TO WRITE:
+> 
+> * distributions are separate from the kubeflow community
+>    * none are currently endorsed or certified
+>    * please reach out to the maintainers to get support
+> * the method for installing can vary from distribution to distribution, please consult the docs for your chosen distribution    
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainers</th></th>
+        <th>Platforms</th>
+        <th>Links</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Kubeflow on AWS</td>
+        <td>AWS</td>
+        <td><a href="https://aws.amazon.com/eks/">Amazon Elastic Kubernetes Service (EKS)</a></td>
+        <td><a href="/docs/distributions/aws/">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Azure</td>
+        <td>Azure</td>
+        <td><a href="https://docs.microsoft.com/azure/aks/">Azure Kubernetes Service (AKS)</a></td>
+        <td><a href="/docs/distributions/azure/">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on GCP</td>
+        <td>Google Cloud</td>
+        <td><a href="https://cloud.google.com/kubernetes-engine">Google Kubernetes Engine (GKE)</a></td>
+        <td><a href="/docs/distributions/gke/">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on IBM Cloud</td>
+        <td>IBM Cloud</td>
+        <td><a href="https://www.ibm.com/cloud/kubernetes-service">IBM Cloud Kubernetes Service</a></td>
+        <td><a href="/docs/distributions/ibm/">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Openshift</td>
+        <td>IBM Cloud</td>
+        <td><a href="https://www.openshift.com/">Openshift</a></td>
+        <td><a href="/docs/distributions/openshift/">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Argoflow</td>
+        <td>Argoflow Community</td>
+        <td>Conformant Kubernetes</td>
+        <td><a href="https://github.com/argoflow/argoflow">Docs</a></td>
+      </tr>
+      <tr>
+        <td>Charmed Kubeflow</td>
+        <td>Canonical</td>
+        <td>Conformant Kubernetes</td>
+        <td><a href="https://charmed-kubeflow.io/docs">Docs</a></td>
```

Comment:
I don't see a huge issue with pointing to external docs here, but, in any case, there is documentation on `distributions` upstream that needs to be maintained. Hence, pointing to internal docs might be beneficial. This:
1. Raises the bar for "becoming a distribution" leading to quality
2. Motivates distribution owners to keep their docs up to date 

> Why is there a need for `distributions` docs upstream? 

Some distributions have it leaving any distribution that does not at a disadvantage.

Reply:
@RFMVasconcelos I have raised an issue to discuss the removal of the distribution-specific docs from kubeflow.org, which will mean the `Getting Started/Installing Kubeflow` page will be the only place which references distributions.

But I agree that if there is currently a doc page on kubeflow.org, we should link there for now.

Reply:
@thesuperzapper ok! Happy to discuss this on a separate thread so we can wrap this higher priority item! :)